### ISSUE TITLE
Can this be a library?

### ISSUE THREAD

**Issue by miraclx** [cyplo/genpass#34]
Would be great if there was a modular interface to this that other applications could depend on.

The CLI interface provided here, therefore being the first.

**Comment by KOLANICH** [comment]
No, it cannot. Even if it was, it is AGPLed, this means it is absolutely useless. You should better use something else, there are plenty of password gens.

**Comment by cyplo** [comment]
Heya ! Thanks for stopping by ! Sorry for late response, I'm not on github that much nowadays.
yeah defo can be a library if you'd want to, let me know if you'd want to pair on this @miraclx .
As for the license, yep, I would like to stick to copyleft ones for this project.

**Comment by cyplo** [comment]
@KOLANICH - btw - I'm really into contributions and making the conversation be friendly, please be mindful of that.

**Comment by miraclx** [comment]
> let me know if you'd want to pair on this @miraclx.

Unfortunately, I don't have as much free time these days. But if you're working on something, I can help review.
